addons/mass_mailing/static/src/js/tours/mass_mailing_editor_tour.js

```diff
@@ -10,16 +10,41 @@ odoo.define('mass_mailing.mass_mailing_editor_tour', function (require) {
         trigger: '.o_app[data-menu-xmlid="mass_mailing.mass_mailing_menu_root"]',
     }, {
         trigger: 'button.o_list_button_add',
-    }, {
-        trigger: 'input[name="subject"]',
-        run: 'text Test',
     }, {
         trigger: 'div[name="contact_list_ids"] .o_input_dropdown > input[type="text"]',
     }, {
         trigger: 'li.ui-menu-item',
     }, {
-        trigger: 'iframe #default',
-        run: () => $('iframe').contents().find('o_editable').html('<p><br/></p>'),
+        trigger: '[name="body_arch"] iframe #empty',
+    }, {
+        trigger: '[name="body_arch"] iframe .o_editable'
```

Comment:
Do we need to click twice on the iframe?

Reply:
The second click is indeed not needed, I'll put an empty function in `run` instead. (it is a verification step to check that the chosen configuration (#empty) finished rendering)